bin/onchange-nfs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep nfs`  ]]
+then
+  echo "nfs is healthy, mounting uploads directory...."
+  mount -t nfs -v -o nolock,vers=3 nfs:/exports /var/www/html/content/uploads
+else
```

Comment:
you could also disable uploads in PHP. Setting the `php_value upload_max_filesize 1` or `0` in the `wp-config.php` (I don't know if `0` is "unlimited") should effectively disable any new uploads. But, it won't disable attempting to edit existing uploads. I'm not sure how big a problem that is.